libs/common/src/common.utils.ts

```diff
@@ -7,17 +7,19 @@ export function paginator<T>(
   current_page: number,
   items_per_page: number
 ) {
-  const page = current_page || 1,
-    per_page = items_per_page || 10,
-    offset = (page - 1) * per_page,
-    paginatedItems = items.slice(offset).slice(0, items_per_page),
-    total_pages = Math.ceil(items.length / per_page);
+  const page: number = Number(current_page) || 1;
```

Comment:
```suggestion
  const page = current_page || 1;
```
won't this work too? because current_page is already set as a number above?

Reply:
This is used to avoid string concatanation